contract-addresses/mainnetAddresses.json

```diff
@@ -77,6 +77,10 @@
     "proxyAdminAddress" : { "artifact" : "ProxyAdmin", "address": "0xf8c2b645988b7658E7748BA637fE25bdD46A704A"},
     "raiAddress": {"artifact" : "IERC20", "address" : "0x03ab458634910AaD20eF5f1C8ee96F1D6ac54919"},
     "raiBondingCurveAddress": { "artifact": "BondingCurve", "address":  "0x25d60212D47Dd8F6Ff0469367E4c6C98Cd3411A5"},
+    "rariPool19DpiAddress": { "artifact" : "CErc20Delegator", "address": "0xf06f65a6b7d2c401fcb8b3273d036d21fe2a5963"},
```

Comment:
Added these copies of the indexCoop and poolParty PCV deposits which adhere to naming conventions